hacks/retail-analytics/QL_OWNER

```diff
@@ -3,3 +3,4 @@ caugusto@google.com
 # Collaborators
 
 ginof@google.com
+meken@google.com
```

Comment:
Please put your name first and it should be just readable names, not email addresses

Reply:
I've changed the order but the email addresses (actually those are account ids) are per Qwiklabs requirements